src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -25,14 +25,14 @@
             [ring.util.response :as rr]
             [schema.core :as schema]
             [slingshot.slingshot :refer [throw+ try+]])
-  (:import clojure.lang.IFn
-           [com.codahale.metrics Gauge MetricRegistry]
-           com.fasterxml.jackson.core.JsonParseException
-           java.io.FileInputStream
-           java.lang.management.ManagementFactory
-           [java.util List Map Map$Entry]
-           org.jruby.exceptions.RaiseException
-           org.jruby.RubySymbol))
```

Comment:
Was this old-style or something? Surprised this works both ways.

Reply:
https://github.com/bbatsov/clojure-style-guide#comprehensive-ns-form

It's not explicit, but all the examples here show it the way it was. Is this change editor managed or is it intentional? If I tell my editor to manage the ns declaration for me it'll put it back to the way it was.

Reply:
the ns form accepts a million different things and i don't think this is actually important, just curious about it

Reply:
It was actually intentional. I originally had an implementation that required a new import and when adding the new import made the section look like https://stuartsierra.com/2016/clojure-how-to-ns.html which is the style I thought we were doing for nses. I meant to at least split this out into its own commit but got lazy. fwiw, I'm happy to put it back.

Reply:
I have no opinion on what style is better or if it even matters to have a consistent one here, I just do the other because it's what emacs does when i tell it to automatically import the things i'm using or to clean up the namespace.